migration/v0.37/prepare/60-move-issuer-ingress-class.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+log_info "moving issuer ingress class keys"
+yq_move common .issuers.letsencrypt.prod.solvers[].http01.ingress.class .issuers.letsencrypt.prod.solvers[].http01.ingress.ingressClassName
+yq_move common .issuers.letsencrypt.staging.solvers[].http01.ingress.class .issuers.letsencrypt.staging.solvers[].http01.ingress.ingressClassName
```

Comment:
Could there be [`extraIssuers`](https://github.com/elastisys/compliantkubernetes-apps/blob/simonl/upgrade-cert-manager/config/config/common-config.yaml#L687) that should be be changed to use `ingressClassName` if they are using the `class` field as well?